### ISSUE TITLE
Missing Wrapper

### ISSUE THREAD

**Issue by drcapulet** [rubygems/rubygems#2686]
### Behavior

When Bundler is installed via a `gem update --system`, the wrapper files normally generated by `gem install` are missing. Notably the `bundle` command is the `exe/bundle` file from the Bundler gem.

### Reproduction

Start image:

```
$ docker run -it --rm ruby:2.5 bash
```

```
root@911c503c3cc4:/# which bundle
/usr/local/bin/bundle
root@911c503c3cc4:/# bundle _1.17.3_ version
Could not find command "_1.17.3_".
root@911c503c3cc4:/# gem install bundler --version 2.0.1
Fetching bundler-2.0.1.gem
Successfully installed bundler-2.0.1
1 gem installed
root@911c503c3cc4:/# hash -r
root@911c503c3cc4:/# which bundle
/usr/local/bundle/bin/bundle
root@911c503c3cc4:/# bundle _1.17.3_ version
Bundler version 1.17.3 (2019-03-04 commit d7089abb6)
root@911c503c3cc4:/#
```

Note that the `hash -r` is required otherwise this will happen:

```
root@e86383a3c9fa:/# which bundle
/usr/local/bin/bundle
root@e86383a3c9fa:/# type bundle
bundle is /usr/local/bin/bundle
root@e86383a3c9fa:/# bundle _1.17.3_ version
Could not find command "_1.17.3_".
root@e86383a3c9fa:/# type bundle
bundle is hashed (/usr/local/bin/bundle)
root@e86383a3c9fa:/# gem install bundler --version 2.0.1
Fetching bundler-2.0.1.gem
Successfully installed bundler-2.0.1
1 gem installed
root@e86383a3c9fa:/# which bundle
/usr/local/bundle/bin/bundle
root@e86383a3c9fa:/# type bundle
bundle is hashed (/usr/local/bin/bundle)
root@e86383a3c9fa:/# bundle _1.17.3_ version
Could not find command "_1.17.3_".
```

**Comment by deivid-rodriguez** [comment]
@drcapulet Thanks for the quality report! :heart:

I agree this is unexpected and I tried to fix it in #2690.

**Comment by drcapulet** [comment]
@deivid-rodriguez thanks for the quick fix!
